library/swift

```diff
@@ -7,8 +7,8 @@ Maintainers: Ted Kremenek <kremenek@apple.com> (@tkremenek),
 GitRepo: https://github.com/apple/swift-docker.git
 
 
-Tags: 4.2.2, 4.2.1, 4.2, 4, latest
-GitCommit: c5dd5e013de66da930cf1b9804797dbae11e8f9c
+Tags: 4.2.3, 4.2.2, 4.2.1, 4.2, 4, latest
```

Comment:
I didn't notice this last time, but it seems a bit odd to call the same image `4.2.3`, `4.2.2` and `4.2.1`.

Reply:
I agree, I noticed this when I took over. I will create a new tag for 4.2.3.

Reply:
I think just dropping `4.2.2` and `4.2.1` will do the trick. Since only `4.2.1`got overwritten by the `4.2.2` image, you could add an extra section with its old `GitCommit` and that should get us a fresh build for that image (and then it can be dropped in a later PR).

Reply:
Updated the tags